### DISCUSSION TITLE
Proposta da criação de um GT de Respostas (CDC)

### DISCUSSION THREAD

**Discussion by anicelysantos** [apyb/comunidade#55] [Geral]
Olá! =)

Participei da Equipe de Resposta esse ano na PyBR e foi minha primeira vez participando de algo desse tipo. Senti a necessidade de algumas coisas durante o processo e é a partir dessa experiência que proponho a criação de um Grupo de Trabalho específico para lidar com o Código de Conduta.

Embora a gente tenha algumas coisas documentadas (os protocolos), acho que seria interessante existir um GT que pudesse garantir que exista uma Equipe de Resposta nos eventos Python e que faça uma formação com essa equipe.

Eu senti um pouco de dificuldade por ter tido pouco tempo de reuniões com a equipe, acho que um GT conseguiria estabelecer um tempo em que essa equipe precisa ser criada e formada para estar pronta no evento.

Com a formação desse GT, algumas outras coisas também poderiam ser padronizadas, como a utilização de uma ferramenta de denúncias (na PyBR 2022 criamos um forms, nas anteriores era usado o email cdc@pythonbrasil).

**Comment by rougeth** [comment]
Obrigado por iniciar essa conversa @anicelysantos!

O código de conduta é um dos pontos mais críticos da organização de um Python Brasil. Pedimos para pessoas voluntárias assumirem um papel de extrema responsabilidade e com muitos poderes (🕷️), sem dar tempo e as ferramentas adequadas para que elas de fato possam fazer um bom trabalho. 

O protocolo que temos foi criado em [2020](https://github.com/pythonbrasil/pybr2020-site/tree/master/src/pdfs/cdc), mas não nunca foi validado na esfera jurídica. Na minha opinião, esse deveria ser o primeiro ponto a ser trabalho pelo grupo, validar se o passo a passo descrito no protocolo embasa as ações realizadas pelo time.

**Reply by anicelysantos** [reply]
Boa! Tá respaldado com o jurídico é a melhor opção.

No contexto de criar o GT, como que funciona? Quais primeiros passos? quem pode participar?
Inclusive, tenho interesse.

**Reply by rougeth** [reply]
> No contexto de criar o GT, como que funciona? Quais primeiros passos? quem pode participar?
Inclusive, tenho interesse.

Ótima pergunta. A APyB não tem um processo formal para criação de grupos de trabalhos, acho que nunca aconteceu antes, e para ser sincero, nem acho que precise de um.

Pensando de forma prática, talvez seja bom ter uma Big Kahuna do GT para intermediar as conversas e demandas com a @apyb/diretoria, mas fora a isso, o grupo tem autonomia para decidir como trabalhar e quem vai participar.

@anicelysantos, como você liderou o time de resposta do CdC da pybr2022, poderia puxar esse GT também!

**Reply by anicelysantos** [reply]
Posso puxar. Não entendi bem o lance de Big Kahuna do GT. Na verdade, ficou um pouco confuso se valeria ter o GT ou não.

Também tenho dúvidas quanto a categoria da minha membresia, se para puxar isso precisaria ser efetivo.

**Reply by rougeth** [reply]
> Posso puxar. Não entendi bem o lance de Big Kahuna do GT. Na verdade, ficou um pouco confuso se valeria ter o GT ou não.

Vale sim! E muito! Não tem burocracia Any, você ta se colocando a disposição para fazer um trabalho importante para a comunidade, fica a vontade para se organizar da melhor forma. A APyB estará aqui para apoiar no que for preciso.

**Comment by cecivieira** [comment]
Também sinto falta de haver um trabalho contínuo voltado para aplicação do Código de Conduta. Acredito que isso tratá amadurecimento nas construções de eventos no qual pense sua construção tendo como pilar o CDC.

Acredito que para dar andamento a construção do GT, poderíamos convocar a primeira reunião através de convite na lista de e-mail.

O que vocês acham da primeira semana de dezembro?

**Comment by AndressaLF** [comment]
Oi pessoal, boa noite.

A **PyNE24** foi meu segundo evento como **Equipe de resposta**, mas foi o meu primeiro como responsável pela mesma.
Nesse processo foi possível viver na pele as dúvidas  que surgem durante a criação do mesmo e os questionamento referente as ações e o papel do CDC após o evento. 

Durante esse processo conversei com @anicelysantos e iniciamos a criação de um documento contemplando esses questionamentos, alguns já possuem respostas escritas ou sugestões do que pensamos que deve ser feito, mas outras perguntas foram apenas registradas com o intuito de serem discutidas em grupo. Por agora esse documento está compartilhado com os membros que fizeram parte da PyNE24(@anapaulamendes @ogecece @carine-neris @lidymonteiro @dandaramcsousa) para que seja construido em conjunto. Essa postagem tem o objetivo de reunir mais nomes para essa contribuição, principalmente de pessoas que já se envolveram com as ações relacionadas ao CDC(@anadulce @biancarosa @cecivieira).
 
A idéia é criar um GT que se dedique a padronizar as ações da Equipe de Resposa, desde sua criação até sua atuação pós evento e além disso permaneca ativo trazendo discussões de melhoria constante junto a APYB e junto a comunidade.

Alguns conversas também já foram levantadas junto a APYB no que diz respeito ao CDC e a atuação da Equipe (@naanadr e @tiidadavena) no que diz respeito a tornar essa tema mais relevante dentro da comunidade visando discussões mais atuais.
